actions/release-fetch/1.0/main.go

```diff
@@ -4,12 +4,21 @@ import (
 	"fmt"
 	"strconv"
 	"time"
-
+	"gopkg.in/yaml.v2"
+	"encoding/json"
 	"github.com/erda-project/erda/apistructs"
 	"github.com/erda-project/erda/pkg/envconf"
 	"github.com/erda-project/erda/pkg/http/httpclient"
 )
 
+type ServiceList struct {
```

Comment:
Please use the 'https://github.com/erda-project/erda/blob/36c63ead20834e0aa75b1f625f5ea0e0eb892a09/pkg/parser/diceyml/diceyml.go' package to parse 'dice.yml'.

Reply:
```shell
diceyml.New -> *DiceYaml 
*DiceYaml.Obj()
```